iocage_lib/ioc_check.py

```diff
@@ -42,21 +43,32 @@ def __init__(self, silent=False, callback=None, migrate=False):
         self.callback = callback
         self.silent = silent
 
+        self.zfs = iocage_lib.ioc_json.IOCZFS()
+        self.pool_mountpoint = self.zfs.zfs_get_property(
+            self.pool, 'mountpoint'
+        )
+        self.iocage_mountpoint = self.zfs.zfs_get_property(
+            f'{self.pool}/iocage', 'mountpoint'
```

Comment:
Doesnt this need to be after check datasets?

Reply:
Oh, yes it indeed should be. Thank you for pointing it out.